services/static-webserver/client/source/class/osparc/desktop/preferences/pages/GeneralPage.js

```diff
@@ -99,15 +93,15 @@ qx.Class.define("osparc.desktop.preferences.pages.GeneralPage", {
       });
       preferencesSettings.bind("creditsWarningThreshold", creditsWarningThresholdField, "value");
       creditsWarningThresholdField.addListener("changeValue", e => this.self().patchPreference("creditsWarningThreshold", creditsWarningThresholdField, e.getData()));
-      form.add(creditsWarningThresholdField, this.tr("Warning threshold"));
+      form.add(creditsWarningThresholdField, this.tr("Show Warning When Credits Below"));
```

Comment:
I would not systematically make the first letter uppercase

Reply:
Not my decision here. I have just added the provided suggestions from app-team.

Reply:
from a English speaking perspective @ignapas is correct it's a sentence and therefore only the first letter should be capital.

Reply:
I Had a chat with EO. He is already aware of the issue and it was already pointed out to the POs. In the future more issues (guidelines) will come about this.

For now with EO, this is what we agreed upon (see screenshot below). Will merge it in this state and when everything is well defined someone will take care of aligning al the texts in the entire application to a well defined standard.


![Screenshot 2023-12-06 at 13 45 34](https://github.com/ITISFoundation/osparc-simcore/assets/5694077/d7cee18a-10cf-4c35-8220-960800ecf059)